scripts/soroban-settings/SorobanSettingsUpgrade.py

```diff
@@ -31,25 +31,85 @@
 
 # hardcode the upgrade you want to do here.
 def get_upgrade_set():
-    state_exp_settings = StateExpirationSettings(max_entry_expiration=Uint32(6312000),
-                                                 min_temp_entry_expiration=Uint32(
-                                                     16),
-                                                 min_persistent_entry_expiration=Uint32(
-                                                     86400),
+
+    max_contract_size = Uint32(65536) # 64 kb
+
+    contract_size_upgrade_entry = ConfigSettingEntry(
+        ConfigSettingID.CONFIG_SETTING_CONTRACT_MAX_SIZE_BYTES,
+        contract_max_size_bytes = max_contract_size)   
+        
+    compute_settings = ConfigSettingContractComputeV0(ledger_max_instructions=Int64(1000000000),
+                                                 tx_max_instructions=Int64(100000000),
+                                                 fee_rate_per_instructions_increment=Int64(100),
+                                                 tx_memory_limit=Uint32(41943040)) # 40 mb
```

Comment:
General comment: we should settle on some multiplier for all the bytes values (either 1024 or 1000). I don't care much which one, as long as it's consistent. Values like `2048000` are inconsistent as they have 1024 * 1000 multiplier, which doesn't make much sense.

If we use 1024 multiplier, I would suggest using it explicitly, like `40 * 1024 * 1024` # 40 MB here.